test.sh

```diff
@@ -59,120 +59,134 @@ fi
 
 AMI_ID=$(aws ec2 describe-images --region $region --filters Name=tag-key,Values=Version Name=tag-value,Values=$TAUPAGE_VERSION --query 'Images[*].{ID:ImageId}' --output  text)
 
-echo "Running AMI tests for $AMI_ID"
-
-create_profile_for_volume_attachment
-create_test_volumes
-sleep 10
-
-# get a server
-echo "Starting test server..."
-result=$(aws ec2 run-instances \
-    --iam-instance-profile Name=${INSTANCE_PROFILE} \
-    --image-id $AMI_ID \
-    --count 1 \
-    --instance-type $instance_type \
-    --associate-public-ip-address \
-    --key-name $keypair \
-    --security-group-ids $security_group \
-    --subnet-id $subnet \
-    --output json \
-    --region $region \
-    --user-data file://$(pwd)/test-userdata.yaml)
-
-instanceid=$(echo $result | jq .Instances\[0\].InstanceId | sed 's/"//g')
-echo "Instance: $instanceid"
+if [ -z $testinstance_types ]; then
+	# backward compability
+	testinstance_types="
+$instance_type
+"
```

Comment:
dunno what brought me to this idea. testinstance_types="$instance_type" is fairly enough

> one thing that I noticed is that the test is then not run using the c3 instances
i added the c3 to the config-stups-exampl.sh
we can also always add it to the list but IMO we than should check for duplicates. So I didn't want to overengineer this.